packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx

```diff
@@ -0,0 +1,263 @@
+import { Meta, Canvas, Story } from "@storybook/addon-docs";
+import { AgGridReact } from "ag-grid-react";
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Grid/AG Grid Theme"
+  component={AgGridReact}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# AG Grid Theme
+
+AG Grid Theme provides CSS theme files for AG Grid. The package contains CSS files only, and should be applicable for all AG Grid versions.
```

Comment:
The first sentence here is a bit weird to me.

Reply:
I agree - have updated it to say "AG Grid Theme gives you the ability to add our theme styles to AG Grid", let me know what you think @joshwooding